x-pack/plugins/monitoring/public/components/table/eui_table_ssp.js

```diff
@@ -72,19 +70,28 @@ export function EuiMonitoringSSPTable({
   const onChange = async ({ page, sort }) => {
     setPage(page);
     setSort({ sort });
-    setIsLoading(true);
-    await fetchMoreData({ page, sort: { sort }, queryText });
-    setIsLoading(false);
-    onTableChange({ page, sort });
+    // angular version
+    if (props.fetchMoreData) {
+      await props.fetchMoreData({ page, sort: { sort }, queryText });
+      onTableChange({ page, sort });
+    }
+    // react version
+    else {
+      onTableChange({ page, sort, queryText });
```

Comment:
fetchMoreData isn't needed in React version because we get data in the ElasticsearchTemplate